.github/workflows/docs.yaml

```diff
@@ -1,9 +1,17 @@
 name: docs
 
 on:
+  pull_request:
   push:
     branches:
       - main
+    paths:
+      - docs/**
+      - docusaurus.config.js
+      - package.json
+      - Vars.js
+      - sidebars.js
+      - yarn.lock
```

Comment:
@nunix I think I got all the files here, is there anything else than can change that affects the docs?